pkg/executor/core/executor_test.go

```diff
@@ -40,6 +41,30 @@ func TestVerifySubject_SubjectParseError(t *testing.T) {
 	}
 }
 
+func TestVerifySubject_ResolveTag_Success(t *testing.T) {
```

Comment:
Would it be possible to add a rag resolve failure case as well?

Reply:
The test above TestVerifySubject_SubjectParseError handles that case where stores doesn't have the tag